### ISSUE TITLE
Documentation security quickstart

### ISSUE THREAD

**Issue by onno-timmerman** [evansd/whitenoise#280]
I was looking today to whitenoise plugin. And reading documentation I notice that in the docs the SecurityMiddleware is commented http://whitenoise.evans.io/en/stable/#quickstart-for-django-apps

This raises the suggestion that you need to comment out the security middleware. Some might make this mistake without being aware.

```

QuickStart for Django apps
Edit your settings.py file and add WhiteNoise to the MIDDLEWARE list, above all other middleware apart from Django’s SecurityMiddleware:

MIDDLEWARE = [
  # 'django.middleware.security.SecurityMiddleware',
  'whitenoise.middleware.WhiteNoiseMiddleware',
  # ...
]
That’s it, you’re ready to go.
```

**Comment by evansd** [comment]
Good point, thanks. Hadn't thought of that. Should probably fix that at some point.

**Comment by adamchainz** [comment]
This was fixed in 704dafa8284959b627e78d43761268e97a7fce87
